modules/planning/scenarios/util/util.cc

```diff
@@ -305,7 +305,7 @@ bool CheckADCHeading(const common::math::Vec2d adc_position,
                      const double adc_heading,
                      const ReferenceLineInfo& reference_line_info,
                      const double heading_diff_to_reference_line) {
-  const double kReducedHeadingBuffer = 0.2;  // TODO(Shu) move to config
+  const double kReducedHeadingBuffer = 0.3;  // TODO(Shu) move to config
```

Comment:
add comments for the units, otherwise people might get confused. (ratio? radiance? degree?)